spdx/file.py

```diff
@@ -22,6 +22,13 @@ class FileType(object):
     BINARY = 2
     ARCHIVE = 3
     OTHER = 4
+    APPLICATION = 5
```

Comment:
looks compliant to the spec:

https://spdx.github.io/spdx-spec/4-file-information/

```
SOURCE if the file is human readable source code (.c, .html, etc.);
BINARY if the file is a compiled object, target image or binary executable (.o, .a, etc.);
ARCHIVE if the file represents an archive (.tar, .jar, etc.);
APPLICATION if the file is associated with a specific application type (MIME type of application/*);
AUDIO if the file is associated with an audio file (MIME type of audio/* , e.g. .mp3);
IMAGE if the file is associated with an picture image file (MIME type of image/*, e.g., .jpg, .gif);
TEXT if the file is human readable text file (MIME type of text/*);
VIDEO if the file is associated with a video file type (MIME type of video/*);
DOCUMENTATION if the file serves as documentation;
SPDX if the file is an SPDX document;
OTHER if the file doesn't fit into the above categories (generated artifacts, data files, etc.)
```